openapi/test.go

```diff
@@ -0,0 +1,36 @@
+package openapi
+
+import (
+	"path"
+	"path/filepath"
+)
+
+// constants and other information used for unit testing.
+// This file serves as a single source of truth for data drawn from in multiple places during testing
+
+const testEndpoint = "/endpoint"
+const testMethod = "post"
+const testReqBody = "RequestBody"
+const testEmptyParamName = "Param"
+const testMediaType = "application/json"
+const openapiFile = "fixtures/validation.yaml"
```

Comment:
Should be prefixed with "test" IMO, as should var declared on line 24

Reply:
done